packages/web3torrent/src/pages/file/File.tsx

```diff
@@ -29,6 +30,8 @@ const File: React.FC<RouteComponentProps> = () => {
   const [errorLabel, setErrorLabel] = useState('');
   const getLiveData = getTorrentAndPeersData(setTorrent, setPeers);
 
+  const channelClient = new ChannelClient(window.channelProvider);
```

Comment:
Could we add a comment here explaining that `window.channelProvider` might be a fake provider (referencing index.tsx)?

Reply:
How would that comment help? As long as the channelProvider implements `ChannelProviderInterface`, `File` shouldn't care if it's fake or real. To me, a comment like that will be out of date once we replace it with real client.